README.md

```diff
@@ -49,6 +49,17 @@ resource "lxd_container" "test1" {
 }
 ```
 
+You can also launch a container directly from a remote image, not locally cached, by referencing the remote image name using the format:
```

Comment:
Could you elaborate on 2. 
In my tests the image was cached locally but without any aliases?!